src/test/java/CarTest.java

```diff
@@ -0,0 +1,107 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CarTest {
+    private Car car = new Car();
+    private static final Double DELTA = 0.0;
```

Comment:
```suggestion
    private static final Double DELTA = 0.00001;
```
краще давати якусь точність, а ніж просто 0.0

Reply:
Done